src/openseadragon.js

```diff
@@ -2453,6 +2453,20 @@ function OpenSeadragon( options ){
             extension = extension ? extension : "";
             // eslint-disable-next-line no-use-before-define
             return !!FILEFORMATS[ extension.toLowerCase() ];
+        },
+
+        /**
+         * Updates supported image formats with user-specified values.
```

Comment:
We should probably include a listing of the default values here. Also it would probably be good to be more explicit about saying that it updates just the specified values, leaving the others unchanged.

Reply:
Added, let me know if it sounds alright!